google-cloud-storage/src/test/java/com/google/cloud/storage/ITStorageLifecycleTest.java

```diff
@@ -66,12 +66,12 @@ public void grpc() throws Exception {
     service1.close();
```

Comment:
When is service2 closed?

Reply:
`service1` and `service2` are the same instance, so when `service1` is closed so it `service2`